packages/sdks/web-js-sdk/src/enhancers/withAutoRefresh/index.ts

```diff
@@ -66,16 +64,15 @@ export const withAutoRefresh =
           return;
         }
         refreshToken = refreshJwt;
-        let timeout =
-          millisecondsUntilDate(sessionExpirationDate) - REFRESH_THRESHOLD;
+        const timeout = getAutoRefreshTimeout(sessionExpirationDate);
+        clearAllTimers();
 
-        if (timeout > MAX_TIMEOUT) {
+        if (timeout <= REFRESH_THRESHOLD) {
```

Comment:
What will happen in this case?
will it break the refresh cycle?

Reply:
added a comment